[2024-09-17T19:01:57.933+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-17T19:01:57.949+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: first_dag.second_task manual__2024-09-17T19:01:55.765528+00:00 [queued]>
[2024-09-17T19:01:57.955+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: first_dag.second_task manual__2024-09-17T19:01:55.765528+00:00 [queued]>
[2024-09-17T19:01:57.956+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 6
[2024-09-17T19:01:57.963+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): second_task> on 2024-09-17 19:01:55.765528+00:00
[2024-09-17T19:01:57.967+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=439) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-09-17T19:01:57.968+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'first_dag', 'second_task', 'manual__2024-09-17T19:01:55.765528+00:00', '--job-id', '50', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpgne2bkbt']
[2024-09-17T19:01:57.969+0000] {standard_task_runner.py:72} INFO - Started process 441 to run task
[2024-09-17T19:01:57.970+0000] {standard_task_runner.py:105} INFO - Job 50: Subtask second_task
[2024-09-17T19:01:58.006+0000] {task_command.py:467} INFO - Running <TaskInstance: first_dag.second_task manual__2024-09-17T19:01:55.765528+00:00 [running]> on host 7c2c6c4cfe77
[2024-09-17T19:01:58.090+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='first_dag' AIRFLOW_CTX_TASK_ID='second_task' AIRFLOW_CTX_EXECUTION_DATE='2024-09-17T19:01:55.765528+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-17T19:01:55.765528+00:00'
[2024-09-17T19:01:58.091+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-17T19:01:58.094+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-09-17T19:01:58.095+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hey am task 2 and will be running after task 1']
[2024-09-17T19:01:58.101+0000] {subprocess.py:86} INFO - Output:
[2024-09-17T19:01:58.102+0000] {subprocess.py:93} INFO - hey am task 2 and will be running after task 1
[2024-09-17T19:01:58.103+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-09-17T19:01:58.121+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-17T19:01:58.122+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=first_dag, task_id=second_task, run_id=manual__2024-09-17T19:01:55.765528+00:00, execution_date=20240917T190155, start_date=20240917T190157, end_date=20240917T190158
[2024-09-17T19:01:58.152+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-17T19:01:58.184+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-17T19:01:58.188+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
